multitenancy.md

```diff
@@ -0,0 +1,94 @@
+# Multitenancy
+
+This document contains sample instructions on how to setup one service cluster together with multiple workload clusters. As stated, the instructions below are just samples, you need update them according to your situation and requirements. You will find more details on the commands executed in the cluster and apps repos.
+
+The following repositores and commits have been tested
+- [ck8s-cluster](https://github.com/elastisys/ck8s-cluster/tree/324ac074a337b99b83a464ce0189ed2c0f267e19)
+- [compliantkubernetes-apps](https://github.com/elastisys/compliantkubernetes-apps)
+
+The cluster setup will be as follows
+- `mtt-0` will be the the repository from where the service cluster will be installed.
+- `mtt-{1,2}` will be repositories from where we'll install workload clusters.
+
+Simply do a _search and replace_ if you want to name your clusters something else.
+`mtt` - simply stands for _multi-tenancy-test_ :)
+
+### Infrastructure and kubernetes
+1. Set some variables
+  ```bash
+  domain=<your_domain>
+  pgp_fp=<your_pgp_fp>
+  config_path=<path_to_store_config_repos>
+  ```
+
+1. Init repos
+  ```bash
```

Comment:
Can you tell how `ck8s_linux_amd64` can be found?

Reply:
In ck8s-cluster when you have built the cli
```
cd ck8s-cluster && make
ls dist/
```